functions/go/gatekeeper/README.md

```diff
@@ -4,29 +4,93 @@
 
 <!--mdtogo:Short-->
 
-Validate the KRM resources using [Gatekeeper] policies.
+The `gatekeeper` function wraps the [`Gatekeeper`] policy enforcement engine to
+evaluate [Gatekeeper] constraints to validate the KRM resources.
 
 <!--mdtogo-->
 
-### FunctionConfig
+It provides a very flexible and powerful way to validate KRM resources against
+custom policies.
+
+You can learn more about how to use the `Gatekeeper` project [here][howto]. The
+concept document about the `OPA Constraint Framework` can be found
+[here][concept].
 
 <!--mdtogo:Long-->
 
-[Gatekeeper] allows users to validate the KRM resources against the Gatekeeper
-policies.
+### Authoring Policies
+
+The `gatekeeper` function follows the [executable configuration] pattern. There
+are 2 kinds of resources needed to define a policy and they need to be provided
+using `input items` along with other KRM resources to be validated.
```

Comment:
`input items` feels like low level detail to me. Can we just say: "and they need to be part of the input resources" ?